### ISSUE TITLE
How to log the Microsoft.Graph.Communications.Common.Telemetry IGraphLogger logs directly into Azure AppInsights?

### ISSUE THREAD

**Issue by himanshupareek** [microsoftgraph/microsoft-graph-comms-samples#455]
Hello Team, I want to know if there is any way to store the Microsoft.Graph.Communications.Common.Telemetry.IGraphLogger logs directly into Azure App-Insights?

**Comment by 1fabi0** [comment]
Yes you can easily write a wrapper with the Application Insights nuget package via IObservable for it and then subscribe the graph logger or create a own Implamentation of the IGraphLogger that also uses Application Insights
